ci/cpu/prebuild.sh

```diff
@@ -6,29 +6,11 @@ set -e
 DEFAULT_CUDA_VER="11.5"
 DEFAULT_PYTHON_VER="3.8"
```

Comment:
I think we can also drop these variables

Reply:
Done